backend/tournesol/tests/test_api_video.py

```diff
@@ -311,9 +311,9 @@ def test_get_video_date_filters(self):
         self.assertEqual(resp.status_code, 200, resp.data)
         self.assertEqual(resp.data["count"], 1)
 
-        resp = client.get("/video/?date_lte=2021-01-03T00:00:00")
+        resp = client.get("/video/?date_lte=2021-01-03T19:45:00")
         self.assertEqual(resp.status_code, 200)
-        self.assertEqual(resp.data["count"], 3)
+        self.assertEqual(resp.data["count"], 2)
```

Comment:
realized this this morning but this test should fail, 3 is expected (19:44:24 < 19:45:00), please do not merge before this is fixed